keyboards/czc/dwarf/readme.md

```diff
@@ -0,0 +1,23 @@
+# CZC.Gaming Dwarf
+
+![czc/dwarf](https://i.imgur.com/AV2fSoy.png)
+
+A keyboard sold by a dommestic reatiler CZC.cz from the Czech Republic.
+
+* Keyboard Maintainer: [Václav Straka](https://github.com/Vesek) (big thanks to the SonixQMK discord)
+* Hardware Supported: CZC.Gaming Dwarf
+* Hardware Availability: [czc.cz official store](https://www.czc.cz/czc-gaming-dwarf-herni-klavesnice-kailh-red-cz/288631/produkt)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make czc/dwarf:default
+
+Flashing example for this keyboard:
+
+    This keyboard must be flashed with an external tool (https://github.com/SonixQMK/sonix-flasher)
```

Comment:
you might want to try the cli variant, SonixFlasherC :)

Reply:
I am using it since the beginning because it's faster to use and also the GUI always SIGFAULTs.
I will modify the readme.md to show a link to both